packages/Checkbox/src/Checkbox.story.tsx

```diff
@@ -2,7 +2,7 @@ import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
 import { storiesOf } from '@storybook/react';
 import { text, boolean } from '@storybook/addon-knobs';
-import { emotion } from '@leafygreen-ui/lib';
+import emotion from '@leafygreen-ui/emotion';
```

Comment:
Looks like not destructuring before was just out of convenience here. We can pull `css` out for this file.

Reply:
👍